### ISSUE TITLE
Add Console Or Log

### ISSUE THREAD

**Issue by deepaspl** [GitFiend/Support#98]
GitFiend is very clean and easy to use.

I show the Console button on start a few process, but it disappears before I clicked on it,
Can you please add a parchment console link or box to check every cmd run in past?

**Comment by GitFiend** [comment]
hi @deepaspl, thanks for the comments. Sorry I didn't respond until now. I intend to improve this over time, but not sure what the right approach is yet.

**Comment by Drasenix** [comment]
maybe a button next to Branches to show a screen|modal with all the git history of current session ?

**Comment by deepaspl** [comment]
yes, that is a good place to start but page is more convenient instead of
model.

thanks.
